packages/p2p-v2/src/components/MobileTabs/MobileTabs.tsx

```diff
@@ -15,12 +15,12 @@ function MobileTabs<T extends string[]>({ onChangeTab, tabs }: TMobileTabsProps<
                 <Button
                     className='p2p-v2-mobile-tabs__tab'
                     color='white'
-                    icon={<LabelPairedChevronRightLgBoldIcon />}
+                    icon={<LabelPairedChevronRightSmRegularIcon />}
                     key={`${tab}-${i}`}
                     onClick={() => onChangeTab(tab)}
                     variant='contained'
                 >
-                    {tab}
+                    <Text size='sm'>{tab}</Text>
```

Comment:
used Text component here because Button doesnt have a prop for handling bold text, and we need this not to be bold